app/build.gradle

```diff
@@ -35,10 +38,13 @@ android {
 dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'androidx.core:core-ktx:1.6.0'
```

Comment:
Why the downgrade?

Reply:
core-ktx v1.7.0 requires sdk version 31. Currently project compiles with version 30.
I've decided to downgrade it. Would've be better if I upgraded compileSdkVersion -> 31?

I think is stable for the purposes of the app as it is.

![image](https://user-images.githubusercontent.com/66312336/140508535-6674e82d-15bc-4250-8b2a-9d0f36d416af.png)